src/Main.purs

```diff
@@ -198,7 +199,9 @@ commands = [
       Args.option "quiet" ["--quiet", "-q"] Type.flag
         "Display nothing to the console when rebuilding."
     ] ++ buildishArgs,
-  Args.command "login" "Obtain and store a token for uploading packages to Pursuit." Nothing Login.action []
+  Args.command "login" "Obtain and store a token for uploading packages to Pursuit." Nothing Login.action [],
+  Args.commandWithArgs "version" "Bump and tag a new version in preparation for release." Nothing BumpVersion.action []
+    [Args.argument "bump" Type.versionBump "The type of bump to perform" false]
```

Comment:
Ah yes, that reminds me, the help is kind of useless for pulp version right now.